packages/react/src/lib/positioner/Popper.tsx

```diff
@@ -0,0 +1,171 @@
+
+  /**
+   * Enables events (resize, scroll).
+   * @prop {Boolean} eventsEnabled=true
+   */
+  eventsEnabled?: boolean
+
+  /**
+   * List of modifiers used to modify the offsets before they are applied to the Popper box.
+   * They provide most of the functionality of Popper.js.
+   */
+  modifiers?: PopperJS.Modifiers
+
+  /**
+   * Enables the Popper box to position itself in 'fixed' mode (default value is position: 'absolute')
+   * @prop {Boolean} positionFixed=false
+   */
+  positionFixed?: boolean
+
+  /**
+   * Ref object containing the target node (the element that we're using as reference for Popper box).
+   */
+  targetRef?: React.RefObject<Element>
+
+  /**
+   * Rtl attribute for the component.
+   */
+  rtl?: boolean
+
+  /**
+   * Array of conditions to be met in order to trigger a subsequent render to reposition the elements.
+   */
+  positioningDependencies?: any[]
+}
+
+/**
+ * Popper relies on the 3rd party library [Popper.js](https://github.com/FezVrasta/popper.js) for positioning.
+ */
+const Popper: React.FunctionComponent<PopperProps> = props => {
+  const {
+    align,
+    children,
+    eventsEnabled,
+    modifiers,
+    offset,
+    pointerRef,
+    position,
+    positionFixed,
+    positioningDependencies,
+    rtl,
+    targetRef,
+  } = props
+
+  const proposedPlacement = getPlacement({ align, position, rtl })
+
+  const popperRef = React.useRef<PopperJS>()
+  const contentRef = React.useRef<HTMLElement>(null)
+  const latestPlacement = React.useRef<PopperJS.Placement>(proposedPlacement)
+  const [computedPlacement, setComputedPlacement] = React.useState<PopperJS.Placement>(
+    proposedPlacement,
+  )
+
+  React.useEffect(
+    () => {
+      const handleUpdate = (data: PopperJS.Data) => {
+        if (data.placement !== latestPlacement.current) {
+          latestPlacement.current = data.placement
+          setComputedPlacement(data.placement)
+        }
+      }
+
+      const pointerRefElement = pointerRef && pointerRef.current
+      const options: PopperJS.PopperOptions = {
+        placement: proposedPlacement,
+        eventsEnabled,
+        positionFixed,
+        modifiers: {
+          ...(offset && {
+            offset: { offset: rtl ? applyRtlToOffset(offset, position) : offset },
+            keepTogether: { enabled: false },
+          }),
+          ...modifiers,
+          arrow: {
+            enabled: !!pointerRefElement,
+            element: pointerRefElement,
+          },
+        },
+        onCreate: handleUpdate,
+        onUpdate: handleUpdate,
+      }
+
+      popperRef.current = new PopperJS(targetRef.current, contentRef.current, options)
+      return () => popperRef.current.destroy()
+    },
+    [eventsEnabled, modifiers, offset, position, positionFixed, rtl, proposedPlacement],
+  )
+
+  React.useEffect(
+    () => {
+      popperRef.current.scheduleUpdate()
+    },
+    [...(positioningDependencies || []), computedPlacement],
```

Comment:
We can `positioningDependencies: []` in props, so we can simplify this:

```suggestion
    [...positioningDependencies, computedPlacement],
```

Reply:
done, thanks